docs/List-of-OSS-Components.md

```diff
@@ -6,19 +6,19 @@ These are the Open Source components used by Microsoft 1DS / C++ Client Telemetr
 
 ZLIB DATA COMPRESSION LIBRARY.
 
-SDK maintains its own snapshot of the mainline ZLib with some Intel architecture performance optimizations [here](https://github.com/microsoft/cpp_client_telemetry/tree/master/zlib).
+SDK maintains its own snapshot of the mainline ZLib with some Intel architecture performance optimizations [here](https://github.com/microsoft/cpp_client_telemetry/tree/main/zlib).
 
 ## [SQLite](https://www.sqlite.org/index.html)
 
 SQLite is a C-language library that implements a small, fast, self-contained, high-reliability, full-featured, SQL database engine.
 
-SDK maintains its own snapshot of the mainline SQLite, which is used for Windows builds [here](https://github.com/microsoft/cpp_client_telemetry/tree/master/sqlite). Other platforms use platform-provided SQLite.
+SDK maintains its own snapshot of the mainline SQLite, which is used for Windows builds [here](https://github.com/microsoft/cpp_client_telemetry/tree/main/sqlite). Other platforms use platform-provided SQLite.
 
 ## [nlohmann/json](https://github.com/nlohmann/json)
 
 JSON for Modern C++.
 
-SDK maintains its own snapshot of the mainline `nlohmann/json` header-only library [here](https://github.com/microsoft/cpp_client_telemetry/blob/master/lib/include/mat/json.hpp).
+SDK maintains its own snapshot of the mainline `nlohmann/json` header-only library [here](https://github.com/microsoft/cpp_client_telemetry/blob/main/lib/include/mat/json.hpp).
```

Comment:
I think many of these links can be changed to relative links?

Reply:
Good point, made them relative now.